src/gameStateService/data-access-objects.ts

```diff
@@ -101,4 +103,13 @@ export const createStonePositionsDAO = ({
 	return randomizedStonePositions;
 }
 
+export const createLevelEndDataDAO = (gameState) => {
+  return {
+		starCount: gameState.levelEndData.starCount,
```

Comment:
Does this need to sit here as a globally shared data or can it be local on the level end scene?